ccswbs/hjckrrh
# [Master] Turn on "Create new revision" by default for all content types (Fixes #180)

Turns on "Create new revision" by default for all content types (Fixes #180)

**Affected content types**
-  events
-  people profiles
-  banners
-  features
-  course outlines
-  faq
-  services
-  webform
-  social

Note that under Default options, "Create new revision" is already turned on for:
- page
- news
- book

# Testing Procedure
- Ensure that all features are in default state.
- Go to Add content, and then select each content type and check that under "URL path settings", the "Create new revision" checkbox is checked.
